lib/plugins/body_reader.js

```diff
@@ -86,12 +87,23 @@ function bodyReader(options) {
         }
 
         function done() {
+            var errorMessage;
             bodyWriter.end();
 
             if (maxBodySize && bytesReceived > maxBodySize) {
                 var msg = 'Request body size exceeds ' +
                     maxBodySize;
-                next(new RequestEntityTooLargeError(msg));
+
+                // Between Node 0.12 and 4 http status code messages changed
+                // RequestEntityTooLarge was changed to PayloadTooLarge
+                // this check is to maintain backwards compatibility
+                if (PayloadTooLargeError !== undefined) {
```

Comment:
I knew there had to be a way to look that up.

Now that I have this in here, I think it might be less brittle to use the Error names, since it actually changed in iojs 3. And if someone is using Restify with iojs 3, we should not break that.